README.md

```diff
@@ -112,13 +112,15 @@ These is the evolving list of TDD requirements which led to the implementation.
 * [x] Money distribution has a precision member, use that instead of the contrived Precision on Rounding.
 * [x] Add unary and binary minus and the division operator.
 * [x] The context has a smallest distribution unit.
+* [x] We can calculate a dimensionless factor by dividing two money objects.
 
 ### Decisions
 
 * We construct `Money` objects only from `decimal` and `int`. The decision how to handle external rounding problems should be done before construction of a `Money` object.
 * We keep Add, Multiply,etc because no all supported frameworks allow default implementations on the interface.
 * We prepare a distribution strategy but do not make it chosable at this point.
 * We support the following operators: unary + and -, and the binary operators ==, !=, +, -, * and /.
+* You can divide to different currencies only with the `Divide(IMoneyExpression, IMoneyExpression, Option<MoneyEvaluationContext>)` method where you have to give a MoneyEvaluationContext with the necessary exchange rates.
```

Comment:
why is MoneyEvaluationContext not in backticks?

Reply:
Fixed in 7e90982d90734be35ebc642edf1918d605151b87